merkle/log_proofs_test.go

```diff
@@ -169,46 +169,38 @@ func TestCalcConsistencyProofNodeAddresses(t *testing.T) {
 	//
 	// For example, the consistency proof between tree size 6 and 7 consists of
 	// nodes i, j and k. The node j is taken instead of its missing parent.
-	var (
-		expectedConsistencyProofFromSize1To2 = []NodeFetch{
+
+	// These should compute the expected consistency proofs.
+	for _, tc := range []struct {
+		size1 int64
+		size2 int64
+		want  []NodeFetch
+	}{
+		{size1: 1, size2: 2, want: []NodeFetch{
 			newNodeFetch(0, 1, false), // b
-		}
-		expectedConsistencyProofFromSize1To4 = []NodeFetch{
+		}},
+		{size1: 1, size2: 4, want: []NodeFetch{
 			newNodeFetch(0, 1, false), // b
 			newNodeFetch(1, 1, false), // h
-		}
-		expectedConsistencyProofFromSize3To7 = []NodeFetch{
+		}},
+		{size1: 6, size2: 7, want: []NodeFetch{
```

Comment:
Is there a reason this was moved up?

Reply:
The order of constants did not match the order in tests, so this is why the diff looks odd. I picked one of the orders. In a follow-up commit it was reordered lexicographically anyway.